libp2p/protocols/rendezvous.nim

```diff
@@ -690,6 +690,15 @@ proc new*(
     minDuration = MinimumDuration,
     maxDuration = MaximumDuration,
 ): T =
+  if minDuration < 1.minutes:
+    raiseAssert("TTL too short: 1 minute minimum")
```

Comment:
I'm not sure raising a Defect here is appropriate.

Reply:
Good catch, you're right, raiseAssert is not the good call here... I did my last review a bit too fast (or too early).